data/yam/agama/auto/lib/base.libsonnet

```diff
@@ -1,6 +1,7 @@
 {
-  bootloader: {
-    stopOnBootMenu: true
+  bootloader(stop_timeout, timeout):: {
```

Comment:
you cannot set both at the same time, they are mutually exclusive, two functions are needed.
The logic below is really strange and this library should work just like a Lego, where you put one piece or the other, avoid as always, conditional logic.

Reply:
updated.